manifest.md

```diff
@@ -239,25 +239,3 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
   }
 }
 ```
-
-# Backward compatibility
-
-The registry will continue to accept uploads of manifests in both the old and new formats.
-
-When pushing images, clients which support the new manifest format SHOULD first construct a manifest in the new format.
-If uploading this manifest fails, presumably because the registry only supports the old format, the client MAY fall back to uploading a manifest in the old format.
-
-When pulling images, clients indicate support for this new version of the manifest format by sending the
-`application/vnd.oci.image.manifest.v1+json` and
-`application/vnd.oci.image.manifest.list.v1+json` media types in an `Accept` header when making a request to the `manifests` endpoint.
-Updated clients SHOULD check the `Content-Type` header to see whether the manifest returned from the endpoint is in the old format, or is an image manifest or manifest list in the new format.
```

Comment:
Does this paragraph fall under “obvious HTTP stuff folks should know from the RFCs without us having to repeat here”?